Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN        bash /tmp/env-config.sh
 # This layer is populated with up-to-date files from
 # Calypso development.
 COPY       . /calypso/
-RUN        npm ci --only=production
+RUN        npm ci --only=production --unsafe-perm
```

Comment:
lerna needed this to be able to run as a postinstall script i think. Need to try removing it and see if things still work.